kola/tests/kubernetes/setup.go

```diff
@@ -82,14 +82,10 @@ func setupCluster(c cluster.TestCluster, nodes int, version, runtime string) (*k
 	}
 
 	// configure nodes via generic install scripts
-	if err := runInstallScript(master, controllerInstallScript, options); err != nil {
```

Comment:
Maybe the remaining error returns in `setupCluster` should also be converted to `Fatal`s, so the function signature doesn't imply that errors will be recoverable?